workflows/mhcquant.nf

```diff
@@ -132,6 +134,25 @@ workflow MHCQUANT {
     RESCORE( ch_rescore_in, ch_multiqc_files )
     ch_versions = ch_versions.mix(RESCORE.out.versions)
     ch_multiqc_files = ch_multiqc_files.mix(RESCORE.out.multiqc_files)
+
+    // GENERATE SPECTRUM LIBRARY
+    if (params.generate_speclib) {
+    OPENMS_COMETADAPTER.out.idxml
```

Comment:
Maybe put this also in the subworkflow?

Reply:
Good catch!

Reply:
My reasoning was that this local subworkflow could potentially go to nf-core, since in general you need only two inputs for the speclib: identification files (FDR-filtered or not) and spectrum files. Therefore I put the channel manipulation before the subworkflow 🤔

Reply:
this subworkflow could also be used in quantms e.g. since DIA-NN needs the spectrum library

Reply:
Very well, then I will approve it